### PR TITLE
Reference PropTypes via prop-types module

### PR DESCRIPTION
**Pull request by hgcummings**
React.Proptypes was deprecated in React v15.5 and removed in React v16

See https://reactjs.org/blog/2017/04/07/react-v15.5.0.html#migrating-from-reactproptypes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,8 +229,8 @@ Additionally, PostCSS makes heavy use of JSON internally, so you will likely nee
 Standard stuff, clone the repo and `npm i` to get the dependencies. npm scripts available:
 * `dev` => runs the webpack dev server for the playground
 * `lint` => runs ESLint against files in the `src` folder
-* `prepublish` => if in publish, runs `prepublish:compileh`
-* `prepublish:compileh` => runs the `lint` and `transpile` scripts
+* `prepublish` => if in publish, runs `prepublish:compile`
```
Comment (hgcummings) [inline README.md:232]: (Drive-by documentation fix)

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "lodash": "^4.16.4",
     "moize": "^2.2.3",
     "postcss": "^5.1.2",
+    "prop-types": "^15.5.7",
```
Comment (hgcummings) [inline package.json:27]: See https://github.com/facebook/prop-types#how-to-depend-on-this-package

> For _libraries_, we also recommend leaving it in `dependencies`

**Comment by planttheidea** [comment]
Good catch! Havent updated this package for a while ... I'll get it out later tonight.
